pkg/render/manager.go

```diff
@@ -470,6 +479,21 @@ func (c *managerComponent) managerProxyProbe() *v1.Probe {
 	}
 }
 
+// managerPacketCaptureLivenessProbe returns the probe for the PacketCapture API container.
+func (c *managerComponent) managerPacketCaptureLivenessProbe() *v1.Probe {
+	return &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path:   "/packetcapture/health",
+				Port:   intstr.FromInt(managerPort),
+				Scheme: corev1.URISchemeHTTPS,
+			},
+		},
+		InitialDelaySeconds: 90,
```

Comment:
These are the default delays we use for all our probes.

Reply:
It would be fine for us to reduce this if we don't have a reason to expect the health check to take a long time to get going (I'm _guessing_ that this one becomes healthy very quickly, so maybe 10s would be fine), but since the manager pod is a singleton I think the impact of this time not being precisely calibrated is almost nothing.

As such I suggest not interfering with the 3.8 schedule for this, we can always make a follow-up PR if you feel strongly.

Reply:
Talked offline. We are not changing the default delays now.